app/networkapi/people/views.py

```diff
@@ -9,7 +12,11 @@ class PeopleListView(ListAPIView):
     A view that permits a GET to allow listing all the People
     in the database
     """
-    queryset = Person.objects.all()
+    now = timezone.now()
```

Comment:
this is going to get unwieldy very soon. Can you follow this pattern instead?
https://github.com/mozilla/api.mozillascience.org/blob/master/app/scienceapi/events/models.py#L8-L21
https://github.com/mozilla/api.mozillascience.org/blob/master/app/scienceapi/events/models.py#L111